examples/hello.rs

```diff
@@ -13,8 +13,15 @@ fn main() -> Result<(), Box<dyn Error>> {
         .build(&event_loop)
         .unwrap();
 
-    let instance = wgpu::Instance::new(wgpu::Backends::all());
-    let surface = unsafe { instance.create_surface(&window) };
+    let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
+        backends: wgpu::Backends::all(),
+        ..Default::default()
+    });
+    let surface = unsafe {
+        instance
+            .create_surface(&window)
+            .expect("Failed to create surface!")
+    };
```

Comment:
This function returns `Result<(), Box<dyn Error>>`. Wouldn't using the `?` operator work?

```suggestion
    let surface = unsafe { instance.create_surface(&window)?};
```